src/backend/routes/onlineLetterRoutes.ts

```diff
@@ -47,14 +49,24 @@ router.post('/read', async (req, res) => {
     res.status(403).json({ error: 'Wrong letter access credentials' });
     return;
   }
+
+  // When recipient opens the reply for the first time, it will update its read receipt to "read".
+  if (reply?.readReceipt === null || reply?.readReceipt === ReadReceiptStatus.unread) {
```

Comment:
Yes, previously I assume that the old records won't get updated when a new column is created, but since there is a default value for the read_receipt this would never be null.